data/tutorials/gs_00_up_and_running.md

```diff
@@ -13,8 +13,9 @@ date: 2021-05-27T21:07:30-00:00
 
 This page will walk you through the installation of everything you need for a comfortable development environment to write projects in OCaml code. Of course, this includes [installing the compiler](#installing-ocaml) itself, but it also installs a build system, a package manager, an LSP server to support your editor, and a few other tools that we describe [later](#setting-up-development-tools), setting up [editor support](#configuring-your-editor), and bootstrapping a [new project](#starting-a-new-project).
 
-The following instructions work on Linux, BSD, and macOS. Plus, they also work
-on Cygwin and WSL. If you want to set up OCaml on Windows, you might be
+We provide installation instructions for Linux, macOS, and *BSD for all OCaml versions. For Windows, we provide instructions on this page for installing OCaml 4.14.0 via the [Diskuv OCaml](https://github.com/diskuv/dkml-installer-ocaml#readme) Installer. Note that, if you use Windows Subsystem for Linux (WSL), the Unix instructions can be used on Windows.
```

Comment:
Not really related to this PR, on this page alone I noticed:

- `Note that, ...`
- `..., please note ...`
- `Note: ...`
- `**Note**` (as a subsection)
- `**Please note:** ...`

I think we should at minima make everything consistent.
Could we consider having special blocks (with special CSS style, like a tooltip box) for this kind of information that breaks the flow of reading the page?

Reply:
That would be nice actually. We could use a custom HTML block for that which we style appropriately.